front/lib/models.ts

```diff
@@ -752,11 +757,18 @@ ChatMessage.init(
       type: DataTypes.TEXT,
       allowNull: true,
     },
+    feedback: {
+      type: DataTypes.STRING,
+      allowNull: true,
+    },
   },
   {
     modelName: "chat_message",
     sequelize: front_sequelize,
-    indexes: [{ fields: ["chatSessionId", "createdAt"] }],
+    indexes: [
+      { unique: true, fields: ["sId"] },
```

Comment:
We expose ourselves to collisions here right? The unique index should be on chatSessionId and sId no?